src/scrape_up/github/__init__.py

```diff
@@ -37,3 +37,15 @@ def get_avatar(self):
         except:
             message = f"Avatart not found for username {self.username}"
             return message
+
+    def get_bio(self):
+        """
```

Comment:
Include this method in the `documentation.md` file.

Reply:
Sure, will do

Reply:
Done updating documentation.md @nikhil25803